### PR TITLE
Adds PR label checks.

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation
1. Enforce that critical labels can only be removed by the jobs admin and members of the release team.
2. Ensures that issue labels don't get added to pull requests.

This PR has been tested at https://github.com/jameesjohn/oppia/pull/26#issuecomment-653815716, https://github.com/jameesjohn/oppia/pull/26#issuecomment-653817424
## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: lib/checkPullRequestLabels.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+const CRITICAL_LABEL_NAME = 'critical';
+const { releaseCoordinators: RELEASE_COORDINATORS, SERVER_JOBS_ADMIN } = require('../userWhitelist.json');
```
Comment (ankita240796) [inline lib/checkPullRequestLabels.js:2]: Is this line too long, if yes please break.

File: userWhitelist.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
 {
+  "releaseCoordinators": [
+    "nithusha21",
+    "Showtim3",
+    "aks681",
+    "U8NWXD",
+    "vojtechjelinek",
+    "kevintab95",
+    "ankita240796",
+    "DubeySandeep"
```
Comment (ankita240796) [inline userWhitelist.json:10]: Update the list to have: nithusha21, aks681, vojtechjelinek, ankita240796, DubeySandeep, BenHenning (Others work as QA co-ordinators.)

File: lib/checkPullRequestLabels.js [hunk 3]
```diff
@@ -149,3 +152,82 @@ module.exports.checkChangelogLabel = async function(context) {
...
+
+  if(label.name === CRITICAL_LABEL_NAME) {
+    // Check that user is allowed to remove the label.
+    const user = context.payload.sender;
+    if (!isUserAllowedToRemoveCriticalLabel(user.login)) {
+      const commentParams = context.issue({
+        body:
+          'Hi @' +
+          user.login +
+          ', only members of the release team are allowed to remove ' +
```
Comment (ankita240796) [inline lib/checkPullRequestLabels.js:182]: Ping @oppia/release-coordinators here as well.

File: lib/checkPullRequestLabels.js (hunk 1 above)
Comment (jameesjohn) [reply lib/checkPullRequestLabels.js]: Done.

File: lib/checkPullRequestLabels.js (hunk 3 above)
Comment (jameesjohn) [reply lib/checkPullRequestLabels.js]: Done.

File: userWhitelist.json (hunk 2 above)
Comment (jameesjohn) [reply userWhitelist.json]: Done.

File: lib/checkPullRequestLabels.js [hunk 4]
```diff
@@ -149,3 +155,83 @@ module.exports.checkChangelogLabel = async function(context) {
...
+  if (!isPrLabel(label)) {
+    // Issue label got added.
+    const user = context.payload.sender;
+    const link = 'here'.link(
+      'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
+        'labeling-issues-and-pull-requests'
+    );
+    const commentBody =
+      'Hi @' + user.login + ', the ' + label.name + ' label should only be ' +
+      'used on issues, and I’m removing the label. You can learn more about ' +
```
Comment (ankita240796) [inline lib/checkPullRequestLabels.js:224]: Just add specifically that refer to point x.

Comment (jameesjohn) [reply lib/checkPullRequestLabels.js:224]: @ankita240796 I'm not sure I understand what you mean here. The link takes them to the exact section on labels to be used on pull requests.

Comment (ankita240796) [reply lib/checkPullRequestLabels.js:224]: Ah ok, sorry I missed that all points are relevant here.

**Pull request merged**